sdk/src/clients/gas/DefaultGasProvider.ts

```diff
@@ -54,7 +50,7 @@ export class DefaultGasProvider implements IEthereumGasProvider<TransactionReque
       };
     }
 
```

Comment:
By revisiting this class, I now think we should change `config.maxFeePerGas` to `maxFeePerGasCap`, this would be aligned with the config meaning which is to set a cap for maxFeePerGas, which means we should also:
- `config.enforceMaxGasFee` to `enforceMaxFeePerGasCap`
- `if (maxPriorityFeePerGas > this.config.maxFeePerGas)` below should change to `if (feeHistory.baseFeePerGas + maxPriorityFeePerGas > this.config.maxFeePerGasCap)`
wdyt? Btw in our envs, for postman, `getGasFees` is used only in L1 to L2 claim and `baseFeePerGas` on L2 should be neglible as its always 7 wei, right?